assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -41,28 +48,43 @@ export const DumbManageModules = ({
       </p>
       <div className="two-columns-admin">
         <div className="column-left">
-          <SelectModulesForm lang={locale} modulesByIdentifier={modulesByIdentifier} toggleModule={toggleModule} />
+          <SelectModulesForm lang={locale} moduleTypes={moduleTypes} modulesById={modulesById} toggleModule={toggleModule} />
+          <div className="margin-xl">
+            <Button className="button-submit button-dark" onClick={() => createIntroductionModule(enabledModules.size - 2)}>
+              <Translate value="administration.landingPage.manageModules.textAndMultimediaBtn" />
+            </Button>
+          </div>
         </div>
         <div className="column-right">
-          <ModulesPreview modules={enabledModulesInOrder} moveModuleDown={moveModuleDown} moveModuleUp={moveModuleUp} />
+          <ModulesPreview modules={enabledModules} moveModuleDown={moveModuleDown} moveModuleUp={moveModuleUp} />
         </div>
       </div>
     </div>
   </div>
 );
 
 const mapStateToProps = (state) => {
-  const { enabledModulesInOrder, modulesByIdentifier } = state.admin.landingPage;
+  const { enabledModulesInOrder, modulesById, modulesInOrder } = state.admin.landingPage;
   return {
-    enabledModulesInOrder: enabledModulesInOrder.map(identifier => modulesByIdentifier.get(identifier)),
-    modulesByIdentifier: modulesByIdentifier
+    enabledModules: enabledModulesInOrder.map(id => modulesById.get(id)),
+    modulesById: modulesById,
+    moduleTypes: modulesInOrder
+      .map((id) => {
+        const module = modulesById.get(id);
+        return module.get('moduleType');
+      })
+      .toJS()
   };
 };
 
 const mapDispatchToProps = dispatch => ({
-  moveModuleDown: identifier => dispatch(moveLandingPageModuleDown(identifier)),
-  moveModuleUp: identifier => dispatch(moveLandingPageModuleUp(identifier)),
-  toggleModule: identifier => dispatch(toggleLandingPageModule(identifier))
+  moveModuleDown: id => dispatch(moveLandingPageModuleDown(id)),
+  moveModuleUp: id => dispatch(moveLandingPageModuleUp(id)),
+  toggleModule: id => dispatch(toggleLandingPageModule(id)),
+  createIntroductionModule: (nextOrder) => {
+    const newId = createRandomId();
+    return dispatch(createLandingPageModules(newId, nextOrder));
+  }
```

Comment:
To avoid renaming later when we implement other module creation, you should maybe name it this way:
```
createModule: (nextOrder, moduleType='INTRODUCTION') => {
  const newId = createRandomId();
  return dispatch(createLandingPageModule(newId, nextOrder, moduleType));
}
```